site/_posts/2017-04-17-Fine-tuning-Fine.md

```diff
@@ -0,0 +1,104 @@
+
+In case you did not know, we base our Release names after some of the really  well known Chess Grandmasters. Chess Grandmasters, by definition, are the Crème De La Crème, and the "IQ" in ManageIQ, well, is all about the IQ. If we are who we associate with, then we surely are in exalted company.
+
+The Chess Grandmaster name that we have chosen for our upcoming F-Release is [The American Chess Grandmaster, Reuben Fine](https://en.wikipedia.org/wiki/Reuben_Fine).
+
+Now let's take a look at some of the **`Finer`** details.
+
+As **`Fine`** has been tailored mostly for the Ansible Provider, we had many Ansible related PRs. 
+So, what is all this **`Fine-tuning`** really about? Well, it's the BUGS! ...Bugs of course.
+
+Said bugs have been the focus nearly all of last week. In addition to the bug resolutions, there also are some refactorings and enhancements.
+
+With the growing number of repositories being split, it's practically impossible to cover all repositories in this article, So...
+
+I'll do my best to bring you some of the **`Fine`** highlights of the repositories that I monitor.
+
+
+## Some Fine 'Finishing' Touches
+
+- [Laura Galis](https://github.com/lgalis) fixed the PDF download issue for Ansible Tower Job Templates in https://github.com/ManageIQ/manageiq-ui-classic/pull/523
+
+- [Martin Hradil](https://github.com/himdel) and [I](https://github.com/AparnaKarve) fixed some synching issues in the Ansible Credential form in 
+
+   https://github.com/ManageIQ/manageiq-ui-classic/pull/860
+ 
+   https://github.com/ManageIQ/manageiq-ui-classic/pull/1017
+
+- [Jillian Tullo](https://github.com/jntullo) worked on a new API endpoint for refreshing Configuration Script Sources
+https://github.com/ManageIQ/manageiq/pull/14714
+
+- [Marcel Hild](https://github.com/durandom) worked on the targeted refresh for Ansible configuration_script_source in 
+https://github.com/ManageIQ/manageiq/pull/14711
+
+- [James Wong](https://github.com/jameswnl) added the Notification for the Ansible Tower credential CUD operations in https://github.com/ManageIQ/manageiq/pull/14625
+
+- [Milan Zázrivec](https://github.com/mzazrivec) added Embedded Ansible refresh for Ansible Credentials and Repositories in https://github.com/ManageIQ/manageiq-ui-classic/pull/934
+
+
+## Some Fine 'Spring-Cleaning'
+
+ We DO love our Spring-cleaning PRs. What say you [Joe Rafaniello](https://github.com/jrafanie)? 
+
+
+- https://github.com/ManageIQ/manageiq/pull/14633/files, courtesy, [Federico Simoncelli](https://github.com/simon3z)
+
+- https://github.com/ManageIQ/manageiq/pull/14598/files, courtesy, [Jozef Zigmund](https://github.com/jzigmund)
+
+- https://github.com/ManageIQ/manageiq/pull/14515/files, courtesy, [Nimrod Shneor](https://github.com/nimrodshn)
+
+- https://github.com/ManageIQ/manageiq/pull/14766/files, courtesy, [Tzu-Mainn Chen](https://github.com/tzumainn)
```

Comment:
@AparnaKarve Is it your intention to refer to the the `/files` section of the PR rather than the main (conversations) part?